site.yml

```diff
@@ -21,460 +21,460 @@ Documents:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Java Client API Reference
+        SEO Description: The MinIO Java Client SDK provides detailed code examples for the Java API
+        SEO Keywords: Java Client SDK, Java Client API
 
       - Python Client Quickstart Guide:
         Link: https://raw.githubusercontent.com/minio/minio-py/master/README.md
         Slug: python-client-quickstart-guide
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Python Client Quickstart Guide
+        SEO Description: The MinIO Python SDK provides simple APIs to access any Amazon S3 compatible object storage server.
+        SEO Keywords: Python SDK, Python Client API
 
       - Python Client API Reference:
         Link: https://raw.githubusercontent.com/minio/minio-py/master/docs/API.md
         Slug: python-client-api-reference
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Python Client API Reference
+        SEO Description: The MinIO Python SDK provides detailed code examples for the Java API
+        SEO Keywords: Python SDK, Python Client API
 
       - Golang Client Quickstart Guide:
         Link: https://raw.githubusercontent.com/minio/minio-go/master/README.md
         Slug: golang-client-quickstart-guide
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Golang Client Quickstart Guide
+        SEO Description: MinIO | Golang Client Quickstart Guide	The MinIO Go Client SDK provides simple APIs to access any Amazon S3 compatible object storage server.
+        SEO Keywords: Golang, Go
 
       - Golang Client API Reference:
         Link: https://raw.githubusercontent.com/minio/minio-go/master/docs/API.md
         Slug: golang-client-api-reference
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Golang Client API Reference
+        SEO Description: The MinIO Go Client API reference provides detailed code examples for the Go API
+        SEO Keywords: Golang, Go
 
       - .NET Client Quickstart Guide:
         Link: https://raw.githubusercontent.com/minio/minio-dotnet/master/README.md
         Slug: dotnet-client-quickstart-guide
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | .NET Client Quickstart Guide
+        SEO Description: The MinIO .NET Client SDK provides simple APIs to access any Amazon S3 compatible object storage server.
+        SEO Keywords: dotnet, .NET
 
       - .NET Client API Reference:
         Link: https://raw.githubusercontent.com/minio/minio-dotnet/master/Docs/API.md
         Slug: dotnet-client-api-reference
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | .NET Client API Reference
+        SEO Description: The MinIO .NET Client SDK provides detailed code examples for the .NET
+        SEO Keywords: dotnet, .NET
```

Comment:
Similar to my previous SDK comments, maybe we can add some keywords similar to `MinIO Dotnet Client` (Not completely sure about capitalization) and/or `MinIO .NET Client`

Reply:
Add don't replace